### PR TITLE
Add perf log level to LoggerPhaseScope ctor

### PR DESCRIPTION
**Pull request by hellosnow**

### CODE REVIEW

File: src/Microsoft.DocAsCode.Common/Loggers/LoggerPhaseScope.cs [hunk 1]
```diff
@@ -18,6 +18,11 @@ public LoggerPhaseScope(string phaseName)
         }
 
         public LoggerPhaseScope(string phaseName, bool enablePerformanceScope)
+            : this(phaseName, enablePerformanceScope, LogLevel.Diagnostic)
+        {
+        }
+
+        public LoggerPhaseScope(string phaseName, bool enablePerformanceScope, LogLevel perfLogLevel)
```
Comment (vwxyzh) [inline src/Microsoft.DocAsCode.Common/Loggers/LoggerPhaseScope.cs]: >bool enablePerformanceScope,  [](start = 50, length = 29)

remove, if use LoggerPhaseScope(phaseName, perfLogLevel), perf log must be true.

Comment (hellosnow) [inline src/Microsoft.DocAsCode.Common/Loggers/LoggerPhaseScope.cs]: @vwxyzh updated, please take a look.

---
In reply to: [94709960](https://github.com/dotnet/docfx/pull/1117#discussion_r94709960) [](ancestors = 94709960)

**Comment by superyyrrzz** [comment]
:shipit:

File: src/Microsoft.DocAsCode.Common/Loggers/LoggerPhaseScope.cs [hunk 2]
```diff
@@ -32,6 +32,12 @@ public LoggerPhaseScope(string phaseName, bool enablePerformanceScope)
             }
         }
 
+        public LoggerPhaseScope(string phaseName, LogLevel perfLogLevel)
```
Comment (ansyral) [inline src/Microsoft.DocAsCode.Common/Loggers/LoggerPhaseScope.cs]: >LoggerPhaseScope [](start = 15, length = 16)

how about the ctor with enablePerformanceScope to call the ctor with perfLogLevel?

**Comment by ansyral** [comment]
:shipit:

**Pull request merged**